packages/packages/api/src/medical/models/link.ts

```diff
@@ -0,0 +1,18 @@
+import { Patient } from "./patient";
+
+export enum MedicalDataSource {
+  COMMONWELL = "COMMONWELL",
+}
```

Comment:
We're defining this twice... and I see that its better to define this on the model layer than external/command. So up to you, if leave this one or move it to a shared file on /medical/models/.., but I suggest we remove the one on external, so we have only one definition

Reply:
Oh man, sorry, I missed the fact this is on the packages.